setup.py

```diff
@@ -12,9 +12,9 @@
 extras= {
     'core': [],
     'tests': ['pytest', 'pytest-mock', 'testresources'],
-    'stable-baselines': ['stable-baselines==2.10.2', "protobuf==3.20.1", "pyyaml"],
-    'stable-baselines3': ['stable-baselines3[extra]==1.6.1', "pyyaml"],
-    'ray-rllib': ['ray[rllib]==2.0.0', 'tensorflow<=2.10.0', 'torch<=1.12.1', "pyyaml"],
+    'stable-baselines': ['stable-baselines==2.10.2', 'gym<=0.21.0', "protobuf==3.20.1", "pyyaml"],
+    'stable-baselines3': ['stable-baselines3[extra]==2.1.0', "pyyaml"],
```

Comment:
Could we relax these constraints now? Something like `'stable-baselines3[extra]=='2.1*'`? Same for rrlib

Reply:
I will with the final version, if no problem will come up, yes!